app/controllers/lookups_controller.rb

```diff
@@ -1,7 +1,12 @@
 class LookupsController < ApplicationController
   def by_base_path
+    # return content_ids for content that is visible on the live site
+    # withdrawn items are still visible
+    states = %w(published unpublished)
```

Comment:
Oh yeah, I was asking more out of general interest for things like the [tagging migration checker](https://github.com/alphagov/finding-things-migration-checker).

I think this request should be able to give you an answer as long as the thing has ever been published.